Lib/asyncio/streams.py

```diff
@@ -1,27 +1,70 @@
 __all__ = (
-    'StreamReader', 'StreamWriter', 'StreamReaderProtocol',
-    'open_connection', 'start_server')
+    'Stream', 'StreamMode',
+    'open_connection', 'start_server',
+    'connect',
+    'StreamServer')
 
+import enum
 import socket
 import sys
 import warnings
 import weakref
 
 if hasattr(socket, 'AF_UNIX'):
-    __all__ += ('open_unix_connection', 'start_unix_server')
+    __all__ += ('open_unix_connection', 'start_unix_server',
+                'connect_unix',
+                'UnixStreamServer')
 
 from . import coroutines
 from . import events
 from . import exceptions
 from . import format_helpers
 from . import protocols
 from .log import logger
-from .tasks import sleep
+from .tasks import sleep, wait
```

Comment:
Can we import `from . import tasks` and then `tasks.sleep`, `tasks.wait`?

Reply:
Done.